### ISSUE TITLE
How to train the images of PASCAL-VOC?

### ISSUE THREAD

**Issue by Li-yachuan** [ZhouCX117/UAED#12]
Hello, I recently replicated UAED on BSDS and the results are consistent with the ones presented in the paper. What a solid work!
But I have a new question, the PASCAL-VOC dataset only has one label per image, how are they trained?
Or are you using a new method of label generation that assigns multiple labels to each image? It seems that more detailed information was not disclosed in the paper.

Looking forward to your reply.

**Comment by ZhouCX117** [comment]
Hi, I have uploaded the pretrain file and checkpoint. We don't generate new labels and simply use the edge label as the std label during the training.

**Comment by Li-yachuan** [comment]
Thank you. I think I understand how to pre-train. But I have a new problem: how do I determine when the pre-training stops? Also get the highest score on the BSDS test set?


---Original---
From: ***@***.***&gt;
Date: Sun, Sep 24, 2023 09:19 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [ZhouCX117/UAED] How to train the images of PASCAL-VOC? (Issue#12)




 
Hi, I have uploaded the pretrain file and checkpoint. We don't generate new labels and simply use the edge label as the std label during the training.

**Comment by ZhouCX117** [comment]
I train 20 epochs and simply choose the last checkpoints.

**Comment by Li-yachuan** [comment]
&nbsp;Thanks for your reply.&nbsp;

---Original---
From: ***@***.***&gt;
Date: Sun, Sep 24, 2023 10:28 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [ZhouCX117/UAED] How to train the images of PASCAL-VOC? (Issue#12)




 
I train 20 epochs and simply choose the last checkpoints.
